nibabel/spatialimages.py

```diff
@@ -866,6 +867,22 @@ def from_image(klass, img):
                      klass.header_class.from_header(img.header),
                      extra=img.extra.copy())
 
+    @classmethod
+    def is_image(klass, filename, sniff=None):
+        ftypes = dict(klass.files_types)
+        froot, ext, trailing = splitext_addext(filename, ('.gz', '.bz2'))
```

Comment:
I'm guessing I made this decision when copying `guessed_image_type` logic in here. I think `klass._compressed_ext` would be fine.